src/react/components/pages/editorPage/canvasCommandBar.tsx

```diff
@@ -64,16 +76,6 @@ export const CanvasCommandBar: React.FunctionComponent<ICanvasCommandBarProps> =
                         isChecked: props.layers["checkboxes"],
                         onClick: () => props.handleLayerChange("checkboxes"),
                     },
-                    // {
-                    //   key: "DrawnRegions",
-                    //   text: strings.editorPage.canvas.canvasCommandBar.items.layers.subMenuItems.drawnRegions,
-                    //   canCheck: true,
-                    //   iconProps: { iconName: "AddField" },
-                    //   isChecked: props.layers["drawnRegions"],
-                    //   className: props.drawRegionMode ? "disabled" : "",
-                    //   onClick: () => props.handleLayerChange("drawnRegions"),
-                    //   disabled: props.drawRegionMode
-                    // },
```

Comment:
Don't delete these. We're leaving them uncommented until back-end is ready

Reply:
add the code back